src/common/models/ResponseState.ts

```diff
@@ -0,0 +1,6 @@
+export enum ResponseState {
```

Comment:
file name convention should be in kebab-case ?

Reply:
Actually, as we talk dashed. 
But somehow, I very like angular naming convention

[feature].[sub_feature].[building_block].ts - worth to think about it

Reply:
renamed to **response-state.enum.ts** what do you think?

Reply:
There is a few files for renaming, if this naming convention is exceptable will go over and rename them